paypalmx.php

```diff
@@ -451,7 +461,17 @@ public function hookDisplayHeader()
 
 	public function hookBackOfficeHeader()
 	{
-		// 2013-11-8 Add 1.4 js and css support
+
+		// Adds 1.5-specific styles
+		if (version_compare(_PS_VERSION_, '1.6', '<')){
+			$css_files = array($this->_path.'css/paypal-mx.css', $this->_path.'css/colorpicker.css', $this->_path.'css/ps15styles.css');
```

Comment:
If you follow the whole function, ```css/paypal-mx.css``` and ```css/colorpicker.css``` will be inserted two times !
This part seems to be executed only on PS 1.5, so you can use addJS and addCSS functions